controls/auth.js

```diff
@@ -7,6 +7,7 @@ var moment = require("moment");
  * route: /user/signup
 */
 exports.postSignUp = function (req, res) {
+    var url = req.url;
```

Comment:
req.url is only supported for requests received through http server
It is not from express
try req.headers.referer
this way you will also get the query parameters